darts/models/forecasting/baselines.py

```diff
@@ -123,6 +123,55 @@ def predict(self, n: int, num_samples: int = 1, verbose: bool = False):
         return self._build_forecast_series(forecast)
 
 
+class NaiveMovingAverage(LocalForecastingModel):
+    def __init__(self, W: int = 1):
```

Comment:
It would be great to have something more meaningful than "W" as argument, something such as "window", "window_size" or "input_chunk_length" to be consistent with other models.

Reply:
Now that you mention it, I have to agree that it could use a better name, for consistency's sake I'll go with  "input_chunk_length"